bentoml/_internal/frameworks/catboost.py

```diff
@@ -84,37 +81,28 @@ def _load_helper(
     return _m
 
 
-@inject
 def load(
     tag: t.Union[str, Tag],
     model_params: t.Optional[t.Dict[str, t.Union[str, int]]] = None,
-    model_store: "ModelStore" = Provide[BentoMLContainer.model_store],
 ) -> "CatBoostModel":
     """
-    Load a CatBoost model from BentoML local modelstore with given name.
-
+    Load a CatBoost model with the given tag from the local BentoML model store.
     Args:
-        tag (:code:`Union[str, Tag]`):
-            Tag of a saved model in BentoML local modelstore.
-        model_params (:code:`Dict[str, Union[str, Any]]`, `optional`, default to :code:`None`): Parameters for
-            a CatBoost model. Following parameters can be specified:
-                - model_type(:code:`str`): :obj:`classifier` (`CatBoostClassifier`) or :obj:`regressor` (`CatBoostRegressor`)
-        model_store (:mod:`~bentoml._internal.models.store.ModelStore`, default to :mod:`BentoMLContainer.model_store`):
-            BentoML modelstore, provided by DI Container.
-
+        tag (``str`` ``|`` :obj:`~bentoml.Tag`):
+            The tag of the model to get from the store.
+        model_params (:code:`Dict[str, Union[str, Any]]`, `optional`, default to :code:`None`): The parameters for loading the CatBoost model. model_type (``str``): can be one of the following:
+                * ``"classifier"`` (``CatBoostClassifier``) 
+                * ``"regressor"`` (``CatBoostRegressor``)
```

Comment:
Hi, please make sure to follow https://github.com/bentoml/BentoML/tree/main/docs#readme

Reply:
I prefer the Union type documentation since we are still supporting python < 3.9. I believe the way you are proposing is the 3.9 syntax?

Reply:
I think I made that change---do you think it's unclear?

Reply:
ye

Reply:
IMO it looks nicer and I don't think it would be confusing but it can be reverted if people don't think we should use the new syntax.

Reply:
@parano @timliubentoml thoughts?

Reply:
I'm a bit bias because I'm not used to python, but I was confused by Union[x,y,z]. I initially thought it was some sort of generic declaration. pipe seems a bit nicer imo